ui/src/app/machines/types.ts

```diff
@@ -22,10 +21,7 @@ export type MachineHeaderContent = HeaderContent<
 
 export type MachineSetHeaderContent = SetHeaderContent<MachineHeaderContent>;
 
-export type MachineActionFormProps = {
-  clearHeaderContent: ClearHeaderContent;
-  errors?: APIError<MachineEventErrors>;
-  machines: Machine[];
-  processingCount: number;
-  viewingDetails: boolean;
-};
+export type MachineActionFormProps = CommonActionFormProps<
+  { machines: Machine[] },
```

Comment:
Is it a bit unnecessary to pass this as a generic? Instead this could be `CommonActionFormProps< MachineEventErrors> & { machines: Machine[] }` (and you'd remove `T extends AnyObject` from the CommonActionFormProps type).

Reply:
Yeah that sounds better tbh, I'm usually not a fan of unnecessary generics so I'm not sure what came over me with this